octoprint_mrbeam/templates/settings/about_settings.jinja2

```diff
@@ -12,11 +12,17 @@
                 <ul>
                     <div><strong>{{ _('Name') }}:</strong> {{ hostname }}</div>
                     <div><strong>{{ _('Serial number') }}:</strong> {{ serial }}</div>
-                    {% if laserhead_serial %}
-                        <div><strong>{{ _('Laser head') }}:</strong> {{ laserhead_serial }}</div>
-                    {% endif %}
-                    {% if laserhead_model == 'S' %}
-                        <div><strong>{{ _('Laser model') }}:</strong> {{ laserhead_model }}</div>
+                    {% if laserhead_serial or laserhead_model == 'S' %}
+                        <div><strong>{{ _('Laser head') }}:</strong>
+                        <ul style="list-style-type: none;">
```

Comment:
- Please avoid using inline CSS

Reply:
I just reused the line from the ethernet section, but can change it